lib/pmp/pmp.go

```diff
@@ -116,7 +116,16 @@ func (w *wrapper) AddPortMapping(ctx context.Context, protocol nat.Protocol, int
 	return port, err
 }
 
-func (w *wrapper) GetExternalIPAddress(ctx context.Context) (net.IP, error) {
+func (*wrapper) AddPinhole(_ context.Context, _ nat.Protocol, _ nat.Address, _ time.Duration) ([]net.IP, error) {
+	// NAT-PMP doesn't support pinholes.
+	return nil, errors.New("adding IPv6 pinholes is unsupported on NAT-PMP")
+}
+
+func (*wrapper) SupportsIPVersion(version nat.IPVersion) bool {
+	return version == nat.IPvAny || version == nat.IPv4Only
```

Comment:
Suggest adding back the comment you previously had on the other routine.

```suggestion
	// NAT-PMP gateways should always try to create port mappings and not pinholes
	// since NAT-PMP doesn't support IPv6.
	return version == nat.IPvAny || version == nat.IPv4Only
```